python/CMakeLists.txt

```diff
@@ -29,15 +29,15 @@ execute_process(
   ERROR_VARIABLE BASIX_ERROR_OUT OUTPUT_STRIP_TRAILING_WHITESPACE
 )
 find_package(Basix REQUIRED CONFIG HINTS ${BASIX_PY_DIR})
+message(STATUS "Found Basix at ${BASIX_DIR}")
```

Comment:
This needs an if/else, e.g.
```
if (Basix_FOUND)
  message(STATUS "Found Basix at ${BASIX_DIR}")
endif()
```

Reply:
Why is that? My understanding is that `find_package(... REQUIRED ...)` will stop with an error if the package is not found, and therefore the line with message will only get printed if the package was actually found.

Reply:
I don't think CMake necessarily stops immediately. It's also a readability issue.

Reply:
OK, will do!